metadata.json

```diff
@@ -1,12 +1,12 @@
 {
-  "name": "genebean-falco",
+  "name": "puppet-falco",
   "version": "0.4.0",
-  "author": "genebean",
+  "author": "puppet",
```

Comment:
Is this right?  I don't think it's just supposed to be a copy of the module's namespace.

Reply:
@genebean @alexjfisher is right. the author needs to be `Vox Pupuli`. Can you please update it?

Reply:
I know some tooling expects it to be the forge username... I can't remember if that is PDK specific or something else though to be honest.